WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -73,14 +73,14 @@ <h4>Accessing my reduced data</h4>
                         The steps below are based on WinSCP as the <code>SFTP Client</code> and assume you have downloaded and installed it.
                         Note that you can only access your data if you are on site, or using the VPN.
                     </p>
-                    <h2>IDAaaS</h2>
+                    <h3>IDAaaS</h3>
```

Comment:
Shouldn't headers that are part of a parent `<h3>` tag be child `<h4>` tags.

For example:

---
### h3 Topic title
content blah ...

#### h4 Topic sub-section
sub content ....

#### h4 Topic sub-section
another sub content ...

---

Reply:
I think that is technically true. I only dropped them because the WAVE tool didnt like how it was originally. I also felt that `h4` was too small and WAVE tool didnt complain about `h3` so I just went with it 😆